add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The HTML and PHP code are mixed, which can lead to maintenance challenges and security issues.

Recommendations:
- Separate the HTML from PHP logic by using a templating engine like Twig or Blade. This will improve the separation of concerns and make the code more maintainable.
- Example of using Twig:
  ```php
  // In your PHP logic
  $loader = new \Twig\Loader\FilesystemLoader('/path/to/templates');
  $twig = new \Twig\Environment($loader);
  echo $twig->render('add_book.html.twig', ['book' => $book]);
  ```
- In `add_book.html.twig`:
  ```html
  <h1>Add a New Book</h1>
  <form method="POST">
      <label>Title:</label>
      <input type="text" name="title"><br>
      <label>Author:</label>
      <input type="text" name="author"><br>
      <label>Year:</label>
      <input type="number" name="year"><br>
      <button type="submit">Add Book</button>
  </form>
  ```